### ISSUE TITLE
OnCreatingTicket not getting called

### ISSUE THREAD

**Issue by mscrivo** [aspnet-contrib/AspNet.Security.OAuth.Providers#269]
I've added LinkedIn auth to our app and while logging in works fine with it, we have another place that we use it to connect to LinkedIn for sharing.  When we call `ChallengeAsync("LinkedIn", properties)` where properties is configured using `signInManager.ConfigureExternalAuthenticationProperties()`, the `OnCreatingTicketEvent` never fires.   I am able to see that `OnTicketReceived` and `OnRedirectToAuthorizationEndpoint` do fire though.

LinkedIn Auth is configured like this:

```C#
services.AddAuthentication
   .AddLinkedIn(options => 
       {
          options.ClientId = hidden;
          options.ClientSecret = hidden;
          options.Scopes.Add("w_share");
          options.Events.OnCreatingTicket = async context =>
              {   // stuff  }
       }
}

**Comment by mscrivo** [comment]
Looks like this has already been fixed in https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/commit/dfd7735216bf3aa6b05ea92823331577f673e214

Is there any workaround until a new nuget package gets published?

**Comment by kinosang** [comment]
If you really need this patched version, you can get the nuget package from the MyGet feed.

**Comment by kevinchalet** [comment]
Patched versions `1.0.1` and `2.0.1` are being pushed to NuGet.org ATM.
